osu.Desktop/LegacyIpc/LegacyIpcMessage.cs

```diff
@@ -48,8 +46,8 @@ public new object Value
 
         public class Data
         {
-            public string MessageType { get; set; }
-            public object MessageData { get; set; }
+            public string MessageType { get; set; } = string.Empty;
+            public object MessageData { get; set; } = default!;
```

Comment:
Not really sure should use `new object()` instead?
The final goal is like https://github.com/dotnet/csharplang/issues/2328 and https://github.com/dotnet/csharplang/issues/3630 to make sure the setter must be assigned the value.
But seems there's no matched syntax for that 😢

Reply:
Can you not make a `ctor` and use that instead?

Reply:
OK, i guess i will use that if there's no better choice 🙆🏼‍♂️

Reply:
Fixed.